<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>serverWindow</class>
 <widget class="QMainWindow" name="serverWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>serverWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab_manage_employee">
     <attribute name="title">
      <string>员工管理</string>
     </attribute>
     <widget class="QTableView" name="tableView_employees_info">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>130</y>
        <width>800</width>
        <height>441</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_manage_ok">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>提交</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>53</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>工号</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_manage_id">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>53</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>密码</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_manage_passwd">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>53</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>姓名</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_manage_name">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>100</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>70</y>
        <width>53</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>生      日</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_manage_birthday">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>70</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>100</y>
        <width>53</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>证件号码</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_manage_id_card">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>100</y>
        <width>140</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>40</y>
        <width>53</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>联系方式</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_manage_tel">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>40</y>
        <width>110</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_manage_entry_time">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>100</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>100</y>
        <width>53</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>入职时间</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>40</y>
        <width>53</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>部门</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_manage_department">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>40</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_manage_position">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>70</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>70</y>
        <width>53</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>职位</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_manage_salary">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>100</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>100</y>
        <width>53</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>工资</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_manage_gender">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>70</y>
        <width>50</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>男</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>女</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>70</y>
        <width>53</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>性      别</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_manage_age">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>70</y>
        <width>50</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>70</y>
        <width>53</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>年龄</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_manage_select_pic">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>40</y>
        <width>140</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>更改员工照片</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_manage_cancel">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_manage_update">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>10</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>刷新</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_manage_select">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_manage_select">
      <property name="geometry">
       <rect>
        <x>275</x>
        <y>11</y>
        <width>400</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_employee_regist">
     <attribute name="title">
      <string>员工注册</string>
     </attribute>
     <widget class="QLabel" name="label_icon">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>10</y>
        <width>450</width>
        <height>450</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(165, 165, 165);</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_regist_age">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>130</y>
        <width>50</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="lineEdit_regist_position">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>280</y>
        <width>140</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_regist_passwd">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>40</y>
        <width>140</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_regist_salary">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>310</y>
        <width>140</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>53</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>姓      名</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>130</y>
        <width>53</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>年龄</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_regist_name">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>70</y>
        <width>140</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_regist_department">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>250</y>
        <width>140</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_regist_tel">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>160</y>
        <width>140</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_regist_gender">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>130</y>
        <width>50</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>男</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>女</string>
       </property>
      </item>
     </widget>
     <widget class="QDateEdit" name="dateEdit_regist_entry_time">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>220</y>
        <width>140</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>130</y>
        <width>53</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>性      别</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_regist_birthday">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>190</y>
        <width>140</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>53</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>工      号</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_regist_id_card">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>100</y>
        <width>140</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>190</y>
        <width>53</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>生      日</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>280</y>
        <width>53</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>职      位</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>53</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>证件号码</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>310</y>
        <width>53</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>工      资</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>160</y>
        <width>53</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>联系方式</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>220</y>
        <width>53</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>入职时间</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>250</y>
        <width>53</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>部      门</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>53</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>密      码</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>470</y>
        <width>451</width>
        <height>26</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btn_open_camera">
         <property name="text">
          <string>打开摄像头</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_close_camera">
         <property name="text">
          <string>关闭摄像头</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_take_a_picture">
         <property name="text">
          <string>拍 照</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>470</y>
        <width>260</width>
        <height>26</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btn_regist_ok">
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_regist_cancel">
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QComboBox" name="comboBox_regist_id">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>140</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_attendance_record">
     <attribute name="title">
      <string>考勤记录</string>
     </attribute>
     <widget class="QTableView" name="tableView_attendance_record">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>39</y>
        <width>800</width>
        <height>541</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_record_select">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>11</y>
        <width>400</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_record_update">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>刷新</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_record_select">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
